docs/guides/advanced-cluster-new-sharding-schema.md

```diff
@@ -1,23 +1,23 @@
 ---
-page_title: "Migration Guide: Advanced Cluster New Sharding Schema"
+page_title: "Migration Guide: Advanced Cluster New Sharding Configurations"
 ---
 
-# Migration Guide: Advanced Cluster New Sharding Schema
+# Migration Guide: Advanced Cluster New Sharding Configurations
```

Comment:
Seeing that in advanced_cluster resource and data source docs we also use `sharding schema` for some example title, would be great if we can adjust for consistency on this change.

Reply:
changes extended to other files